entity-management-web/src/test/java/eu/europeana/entitymanagement/config/LanguageCodesTest.java

```diff
@@ -0,0 +1,48 @@
+package eu.europeana.entitymanagement.config;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+
+import eu.europeana.entitymanagement.common.config.AppConfigConstants;
+import eu.europeana.entitymanagement.common.config.LanguageCodes;
+
+/**
+ * JUnit test to check if DataSources are properly deserialized from XML
+ */
+@SpringBootTest()
```

Comment:
This test will fail once #54 is merged, as [EnrichmentDao](https://github.com/europeana/entity-management/pull/54/files#diff-4cad98fd738004688cac084e0c25db3523e9dfcab0183e5b1e3ac1cf8945fb47R30) connects to Mongo on startup (unlike the lazy connections we initiate in the existing data sources).

Specify component classes relevant for this test, instead of loading the entire application context.

Reply:
For this test the SerializationConfig is sufficient, I updated the code